### DISCUSSION TITLE
Magic Forms for octobercms 2

### DISCUSSION THREAD

**Discussion by GerardoNa** [skydiver/october-plugin-forms#261] [General]
I tried to install Magic Forms in the new version of October so I get an error: "SQLSTATE [42S02]: Base table or view not found: 1146 Table 'dboctober.martin_forms_records' does not exist (SQL: select count (*) as an aggregate of martin_forms_records where unread = 1 and martin_forms_records.deleted_at is null) ".

Could it be that this magnificent complement cannot be used in the new version of October?

**Comment by rubenvanerk** [comment]
It [seems](https://github.com/skydiver/october-plugin-forms/discussions/251#discussioncomment-751355) that the author is not interested in supporting October CMS v2.

**Reply by eddycodes** [reply]
Is there a fork with October 2 support? This is an essential plugin. It's a shame it's not supported

**Comment by GerardoNa** [comment]
I managed to install it by console, you need to run:

php artisan plugin: install martin.forms

when I finish running:

php artisan october: migrate

Ready.
